cartesian_controllers/test/test_twist_controller.cpp

```diff
@@ -43,7 +43,7 @@ TEST_F(TwistControllerTest, joint_names_parameter_not_set)
   ASSERT_TRUE(controller_->joint_name_.empty());
   ASSERT_TRUE(controller_->interface_names_.empty());
 
-  controller_->get_node()->set_parameter({"joint", joint_name_});
+  controller_->get_node()->set_parameter({"interface_names", interface_names_});
```

Comment:
This test was called `joint_names_parameter_not_set` but was setting the joint param.
The following test was called `interface_parameter_not_set` but was setting interface_name parameter.

These two tests just check to it isn't possible to configure the controller with missing mandatory parameters.

Reply:
That is correct, could be that additional work was required.